packages/devtools_app/test/provider/provider_controller_test.dart

```diff
@@ -2,27 +2,29 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe
-
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:devtools_app/devtools_app.dart';
 import 'package:devtools_app/src/screens/provider/instance_viewer/instance_details.dart';
 import 'package:devtools_app/src/screens/provider/instance_viewer/instance_providers.dart';
 import 'package:devtools_app/src/screens/provider/provider_nodes.dart';
 // ignore_for_file: implementation_imports, invalid_use_of_visible_for_testing_member, non_constant_identifier_names
 
 import 'package:devtools_app/src/shared/eval_on_dart_library.dart';
-import 'package:devtools_app/src/shared/globals.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:vm_service/vm_service.dart' hide SentinelException;
 
+// ignore: import_of_legacy_library_into_null_safe
 import '../test_infra/flutter_test_driver.dart';
+// ignore: import_of_legacy_library_into_null_safe
```

Comment:
instead of repeating `// ignore: import_of_legacy_library_into_null_safe`, just revert back to using ignore_for_file at the top of the file

Reply:
It seems that all the ignores could be removed after-all. So I did that instead